flood_adapt/dbs_controller.py

```diff
@@ -1149,15 +1149,11 @@ def delete_scenario(self, name: str):
             )
         else:
             scenario_path = self.input_path / "scenarios" / name
-            scenario = Scenario.load_file(scenario_path / f"{name}.toml")
-            scenario.init_object_model()
-            if scenario.direct_impacts.hazard.has_run:
-                # TODO this should be a check were if the scenario is run you get a warning?
-                raise ValueError(
-                    f"'{name}' scenario cannot be deleted since the scenario has been run."
-                )
-            else:
-                shutil.rmtree(scenario_path, ignore_errors=True)
+            shutil.rmtree(scenario_path, ignore_errors=False)
+
+            results_path = self.input_path.parent / "output" / "Scenarios" / name
+            if results_path.exists():
```

Comment:
shouldn't there be a warning here that we can use in the front end, in case the scenario has already been run?